Created all seven required functions, header, runner, makefile file, …

ivanovmp opened the pull request:
…readme.markdown.

ivanovmp committed:
Created all seven required functions, header, runner, makefile file, readme.markdown.

GassaFM commented on restricted_growth_string/restricted_growth_string.cpp:
```diff
@@ -0,0 +1,183 @@
+
+vector<vector<int64_t>> RestrictedGrowthString::d = initialize_d();
+RestrictedGrowthString::~RestrictedGrowthString(){}
+CombinatorialObject::~CombinatorialObject(){}
```
no newline at end of file

GassaFM commented on restricted_growth_string/restricted_growth_string.h:
```diff
@@ -0,0 +1,46 @@
+	/// destructor
+	~RestrictedGrowthString();
+};
+#endif // RESTRICTED_GROWTH_STRING_H_
```
no newline at end of file

GassaFM commented on restricted_growth_string/restricted_growth_string.h:
```diff
@@ -0,0 +1,46 @@
+	bool is_valid(vector<int> const& v);
+
+	/// 0-based lexicographical number of given valid object,
+	/// undefined if overflow
```
то есть этот балл не предполагается?

GassaFM reviewed (changes requested):
combinatorial-objects.sdf восьмимегабайтный бинарник - зачем? Неплохо бы обойтись без него.

ivanovmp committed:
Added two newlines at the ends of the files, fixed one comment about my
incredible achievement in returning INT64_MAX, not undefined value, in
case of overflow, and removed 8-MB binary file not only fron repository,
but even from my damned harddrive! I'm not even able now to compile it
with my Visual Studio! Goodness gracious, thank you GIT.

ivanovmp committed:
Added #include<string>, removed a dectructor, made this compilable with
vizhak.

ivanovmp committed:
Fixed .git, fixed combinatorial_object

ivanovmp committed:
Removed two annoying files.

GassaFM commented:
Уже почти всё хорошо, но бинарник до сих пор не удалён насовсем из всех версий. Предлагаю закрыть pull request, в своём репозитории снести его ([вот инструкция](https://help.github.com/articles/removing-sensitive-data-from-a-repository/)), и открыть новый pull request с той версией, которая получится. Диагностика - директория .git должна занимать не мегабайты, а десятки-сотни килобайт.
